src/connectors/configure-related-items/connectConfigureRelatedItems.ts

```diff
@@ -148,8 +141,7 @@ See https://www.algolia.com/doc/api-reference/api-parameters/optionalFilters/
     const makeConfigure = connectConfigure(renderFn, unmountFn);
 
     return {
-      ...makeConfigure({ searchParameters }),
-
+      ...makeConfigure({ searchParameters } as any),
```

Comment:
```suggestion
      // required, since widget parameters differ between these connectors
      // and we don't want to have the parameters of configure here
      ...makeConfigure({ searchParameters } as any),
```